OmsAgent/omsagent.py

```diff
@@ -1021,17 +1021,11 @@ def exit_if_openssl_unavailable(operation):
 def exit_if_gpg_unavailable(operation):
     """
     Check if gpg is available to use
-    If not, attempt to install
-    If install fails, throw error to return UnsupportedGpg error code
+    If not found, throw error to return UnsupportedGpg error code
     """
     check_exit_code, _ = run_get_output('which gpg', True, False)
     if check_exit_code is not 0:
-        hutil_log_info('GPG not found, attempting to install')
```

Comment:
could you check extension install telemetry to see if this is a common thing that we do on distros other than debian 10? want to make sure that this isn't a common scenario, because if it is this could be a big regression in install success for customers

Reply:
@hestolz , this is a code Ashley added just recently specifically to enable Debian 10 but it currently has issues and needs to be tested with more distros before allowing it in extension. My change is reverting the install logic till more testing is done.

Reply:
great, thanks for the context, i had been under the impression this was a long-standing chunk of code